mod_reforged/hooks/config/strings.nut

```diff
@@ -1133,8 +1140,8 @@ foreach (vanillaDesc in vanillaDescriptions)
 		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"Hits to the head with one-handed maces apply the [Dazed|Skill+dazed_effect] effect for 1 [turn|Concept.Turn] and if the target is already [dazed,|Skill+dazed_effect] apply [Stunned|Skill+stunned_effect] for one [turn.|Concept.Turn]",
-				"Hits to the head with two-handed maces apply the [Stunned|Skill+stunned_effect] for 1 [turn.|Concept.Turn]"
+				"Hits to the head with one-handed maces apply the [Dazed|Skill+dazed_effect] effect for 1 [turn|Concept.Turn] and if the target is already [Dazed,|Skill+dazed_effect] apply [stunned|Skill+stunned_effect] for one [turn.|Concept.Turn]",
```

Comment:
I prefer the `dazed` here with lowercase as it flows in the sentence better imo.